generators/server/index.js

```diff
@@ -706,16 +716,6 @@ module.exports = JhipsterServerGenerator.extend({
             }
         },
 
-        writeServeri18nFiles: function () {
-
-            // install all files related to i18n if translation is enabled
-            if (this.enableTranslation) {
-                this.installI18nResFilesByLanguage(this, SERVER_MAIN_RES_DIR, 'en');
-                this.installI18nResFilesByLanguage(this, SERVER_MAIN_RES_DIR, 'fr');
-            }
-            this.template(SERVER_MAIN_RES_DIR + 'i18n/_messages_en.properties', SERVER_MAIN_RES_DIR + 'i18n/messages.properties', this, {});
```

Comment:
The language generation will be called from the app index.js with the skip client flag